composer.json

```diff
@@ -11,12 +11,12 @@
         }
     ],
     "require": {
-        "php": "^7.1",
+        "php": ">=7.1",
```

Comment:
i'd prefer to not do this, it might create hassles when php 9 comes out. 

lets do ^7.1 || ^8.0

or 7.2 or whatever. i think the rest of the php-http currently supports 7.2

can we allow some older phpspec version that works on 7.2? there seem to have been no changes needed to the spec files, so it should be cheap to allow it.

Reply:
Sure, there’s quite some discussion about this still: https://twitter.com/nicolasgrekas/status/1263023258938548225.

I’m okay with narrowing the version constraint, but decided in favour of 7.3, which is the earliest supported PHP version.